packages/minter-contracts/ligo/src/common.mligo

```diff
@@ -0,0 +1,57 @@
+(*TYPES*)
+
+type fee_data = 
+  [@layout:comb]
+  {
+    fee_address : address;
+    fee_percent : nat;
+  }
+
+type fa2_token =
+  [@layout:comb]
+  {
+    token_id : token_id;
+    amount : nat;
+  }
+type tokens =
```

Comment:
perhaps a better name would be `fa2_tokens` to make it consistent with `fa2_token`